test/TestGrains/ValueTypeTestGrain.cs

```diff
@@ -1,43 +0,0 @@
-﻿using System.Threading.Tasks;
-using Orleans;
-using UnitTests.GrainInterfaces;
-
-namespace UnitTests.Grains
-{
-    public class ValueTypeTestGrainState
-    {
-        public ValueTypeTestData StateData { get; set; }
-    }
```

Comment:
This class `ValueTypeTestGrainState` is no longer needed. This used to inherit from GrainState, but that's no longer needed, and for some reason that inheritance was removed at the same time the file was moved (hmmm), so that's why diffing it was hard and the change was missed.